crates/cli/tests/install.rs

```diff
@@ -131,3 +131,28 @@ fn should_install_index_files() {
 
     drop((root, mock_instance)); // cleanup
 }
+
+#[test]
+fn should_install_circular_dependencies() {
+    let CommandTempCwd { pacquet, root, workspace, npmrc_info, .. } =
+        CommandTempCwd::init().add_mocked_registry();
+    let AddMockedRegistry { store_dir, mock_instance, .. } = npmrc_info;
+
+    eprintln!("Creating package.json...");
+    let manifest_path = workspace.join("package.json");
+    let package_json_content = serde_json::json!({
+        "dependencies": {
+            "@pnpm.e2e/circular-deps-1-of-2": "1.0.2",
+        },
+    });
+    fs::write(&manifest_path, package_json_content.to_string()).expect("write to package.json");
+
+    eprintln!("Executing command...");
+    pacquet.with_arg("install").assert().success();
+
+    eprintln!("Snapshot");
+    let index_file_contents = index_file_contents(&store_dir);
+    insta::assert_yaml_snapshot!(index_file_contents);
```

Comment:
do really need this snapshot test? Maybe just check if the package is present in `node_modules`.

Reply:
done ~